lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -68,7 +68,7 @@ export class AuthorizationCodeClient extends BaseClient {
         );
 
         // set the final cache and return the auth response
-        this.updateCache();
+        await this.updateCache();
```

Comment:
@jasonnutter  I would love a TS rule that doesn't allow floating un-awaited promises. Its sorta hard to make sure I got all the async stuff that needs to be awaited. I looked into it but couldnt figure it out